dependencyManager.js

```diff
@@ -76,6 +76,10 @@ function getRequiredDeps(packageJson) {
         Object.assign(deps, { "awesome-typescript-loader": "~3.1.3" });
     }
 
+    if (isSass({packageJson})) {
```

Comment:
Don't we need also `node-sass`?

Reply:
No. It turns out node-sass is a dep to sass-loader. :)